drivers/sensor/am2301b/am2301b.c

```diff
@@ -0,0 +1,188 @@
+/* am2301b.c - Driver for Aosong AM2301B Humidity and Temperature Module */
+/*
+ * Copyright (c) 2022 Hisanori Yamaoku
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT aosong_am2301b
+
+#include <device.h>
+#include <devicetree.h>
+#include <drivers/i2c.h>
+#include <drivers/sensor.h>
+#include <logging/log.h>
+
+#ifdef CONFIG_AM2301B_CRC_CHECK
+#include <sys/crc.h>
+#endif /* CONFIG_AM2301B_CRC_CHECK */
+
+#include "am2301b.h"
+
+LOG_MODULE_REGISTER(AM2301B, CONFIG_SENSOR_LOG_LEVEL);
+
+#define AM2301B_LEN_MEASUREMENT_DATA              7U
+
+const static uint8_t am2301b_cmd_initialize[] = { 0xBE, 0x08, 0x00 };
+const static uint8_t am2301b_cmd_measurement[] = { 0xAC, 0x33, 0x00 };
```

Comment:
nit: please document these constants as other areas of the code seem to do

Reply:
fixed it.